### PR TITLE
feat(StatusQ/Controls): introducing StatusBackButton

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes: https://github.com/status-im/status-desktop/issues/6727

### Checklist

- [x] follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
  - the scope should be the component's name e.g: `feat(StatusListItem): ... `
  - when adding new components, the scope is the module e.g: `feat(StatusQ.Controls): ...`
- [ ] add documentation if necessary (new component, new feature)
- [x] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [x] test changes in both light and dark theme?
- [ ] is this a breaking change?
    - [ ] use the dedicated `BREAKING CHANGE` commit message section
    - [ ] resolve breaking changes in [status-desktop](https://github.com/status-im/status-desktop)
        - [ ] (pre-merge) adapt code to breaking changes
        - [ ] (post-merge) update StatusQ submodule pointer
- [x] test changes in [status-desktop](https://github.com/status-im/status-desktop)

### CODE REVIEW

**Review by PascalPrecht — APPROVED** [review]
Yay!!

We should probably a search through the entire codebase (also of desktop) for `arrow-right` and check if there's other places where this happens

**Review by caybro — APPROVED** [review]

File: src/StatusQ/Popups/StatusStackModal.qml [hunk 1]
```diff
@@ -55,8 +55,6 @@ StatusModal {
     leftButtons: StatusRoundButton {
         id: backButton
         icon.name: "arrow-right"
-        icon.width: 20
-        icon.height: 16
         rotation: 180
```
Comment (caybro) [inline src/StatusQ/Popups/StatusStackModal.qml:58]: Just curious... don't we have a "arrow-left" icon? And secondly, maybe this "Back" button is worth a separate/reusable component?

Comment (PascalPrecht) [reply src/StatusQ/Popups/StatusStackModal.qml:58]: > Just curious... don't we have a "arrow-left" icon? 

We do by now: https://github.com/status-im/StatusQ/blob/master/src/assets/img/icons/arrow-left.svg

>  maybe this "Back" button is worth a separate/reusable component?

Yup

**Review by MishkaRogachev — APPROVED** [review]

**Comment by alexandraB99** [comment]
@PascalPrecht @caybro I added the back button, please check and let me know any comments

**Comment by PascalPrecht** [comment]
@alexandraB99 Nice!

Once https://github.com/status-im/StatusQ/pull/821/commits/3ed4dc8b829bd32d07e2a3c845564cc70fe6a0dd is a feature commit, this is gtg!

**Pull request merged**